src/GF.jl

```diff
@@ -32,8 +32,10 @@ export computegf
 """
 function computegf(ed::EDCore,
                    grid::TimeGrid,
-                   c_cdag_index_pairs::Vector{Tuple{IndicesType, IndicesType}},
-                   β)
+                   c_cdag_index_pairs::Vector{Tuple{IndicesType, IndicesType}})
+
+  β = length(get_branch(grid.contour, imaginary_branch))
```

Comment:
The very reason I introduced the `β` argument is because `grid.contour` can be a two-branch contour which does not contain the Matsubara branch at all. In principle, it still makes sense to have a thermal initial state even when one is interested only in the real-time components of a GF.

I think a better approach here is to define an overload of `computegf()`, which tries to extract `β` from the contour and throws an exception if `get_branch()` returns `nothing`.